README.md

```diff
@@ -40,3 +41,47 @@ In your eslint.json file enable this rule with the following:
     }
 }
 ```
+
+## Advanced configuration
+
+```js
+{
+    "plugins": ["no-unsanitized"],
+    "env": {
+        // $ruleName is either "method" or "property"
+        "no-unsanitized/$ruleName": ["errorlevel",
```

Comment:
I prefer giving clear examples over making our users do the variable parsing.
But let's leave the README update out of this and discuss in another patch. I'm even happy to take that one.